docs/changelog.adoc

```diff
@@ -106,7 +106,7 @@ quartz-1.7.3 	24-Feb-2010
 quartz-1.7.2 	10-Feb-2010 
 ----
 
-See https://jira.terracotta.org/jira/browse/QTZ/?selectedTab=com.atlassian.jira.jira-projects-plugin:changelog-panel[older JIRA for older release changelog].
+See https://jira.terracotta.org/jira/browse/QTZ/?selectedTab=com.atlassian.jira.jira-projects-plugin:changelog-panel[older Jira for older release changelog].
```

Comment:
Jira rebranded a while ago, and Atlassian products will actively change `JIRA` to `Jira`